test/extensions/alert_extension.unit

```diff
@@ -0,0 +1,86 @@
+>>> type note
+> [!NoTe]
```

Comment:
These tests look great! Can you see about this text: `\[!note]` or `[\!note]` and see what GitHub does and ensure we do the same. Other than that, test cases look great!

Reply:
> \[!note]
> foobar

Ah it shows an alert. I'll add this case.